employee_seniority_level/tests/:bd

```diff
@@ -0,0 +1 @@
+from . import test_employee_security_level
```

Comment:
this to be removed?

Reply:
Yes thanks, how did this file appear ? It looks like a vim command ;)
Removed